developer/cmdlet/defining-default-methods-for-objects.md

```diff
@@ -39,7 +39,7 @@ In the following example, the **ConvertLargeIntegerToInt64** method is added to
 
 ## Script Methods
 
-A script method defines a method whose value is the output of a script. In the following example, the **ConvertToDateTime** method is added to the [System.Management.Managementobject?Displayproperty=Fullname](/dotnet/api/System.Management.ManagementObject) type. The [ScriptMethod](http://msdn.microsoft.com/en-us/59f8160f-bc95-42f0-92e2-b16a616bc65c) element defines the extended method as a script method. The [Name](http://msdn.microsoft.com/en-us/b58e9d21-c8c9-49a5-909e-9c1cfc64f873) element specifies the name of the extended method. And, the [Script](http://msdn.microsoft.com/en-us/1937ad1b-bb2b-4512-9864-01fc0767d46f) element specifies the script that generates the method value. (You can also add the [ScriptMethod](http://msdn.microsoft.com/en-us/59f8160f-bc95-42f0-92e2-b16a616bc65c) element to the members of the [MemberSets](http://msdn.microsoft.com/en-us/46a50fb5-e150-4c03-8584-e1b53e4d49e3) element.)
+A script method defines a method whose value is the output of a script. In the following example, the **ConvertToDateTime** method is added to the [System.Management.Managementobject?Displayproperty=Fullname](/dotnet/api/System.Management.ManagementObject) type. The [ScriptMethod](https://msdn.microsoft.com/59f8160f-bc95-42f0-92e2-b16a616bc65c) element defines the extended method as a script method. The [Name](https://msdn.microsoft.com/b58e9d21-c8c9-49a5-909e-9c1cfc64f873) element specifies the name of the extended method. And, the [Script](https://msdn.microsoft.com/1937ad1b-bb2b-4512-9864-01fc0767d46f) element specifies the script that generates the method value. (You can also add the [ScriptMethod](https://msdn.microsoft.com/59f8160f-bc95-42f0-92e2-b16a616bc65c) element to the members of the [MemberSets](https://msdn.microsoft.com/46a50fb5-e150-4c03-8584-e1b53e4d49e3) element.)
```

Comment:
The link of ScriptMethod https://msdn.microsoft.com/59f8160f-bc95-42f0-92e2-b16a616bc65c is forwarded to the sorry page.

Reply:
Yup, looks like it's broken with or without the locale